### PR TITLE
added guide

### PR DESCRIPTION
**Pull request by mperkins808**

### CODE REVIEW

**Comment by ghost** [comment]
Mat, this is amazing!
Could this also post this into the handbook under the training folder?

**Comment by ginge270** [comment]
> Matt, this is amazing!
> Could this also post this into the handbook under the training folder?

I second this! Worth also flagging to the documentation team

File: user_guides/configuringJenkins.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# How to download and install Jenkins
+
+## Why Jenkins 
+
+Based on our recent meeting with Andrew, we have decided that the best solution is to create a dedicated CI/CD tool because only using built in GCP tools is not user friendly for future students. It is best for all deployments to GCP be centralised. I believe Jenkins is probably the best solution as it is widely adopted, has GCP plugins and is OSS. I believe that success will follow these steps
```
Comment (ginge270) [inline user_guides/configuringJenkins.md]: The best solution to what?

 I think you can also remove the part about Andrew. Think about this documentation as for someone coming in with no context. What do they need to know? What is GCP?

Comment (mperkins808) [reply user_guides/configuringJenkins.md]: Ok will do. I more made it specifically for our team so I thought the context was important. Will remove.

**Pull request merged**